frontend/src/app/features/work-packages/services/notifications/work-package-notification.service.ts

```diff
@@ -53,14 +60,26 @@ export class WorkPackageNotificationService extends HalResourceNotificationServi
 
   protected showCustomError(errorResource:any, resource:WorkPackageResource):boolean {
     if (errorResource.errorIdentifier === 'urn:openproject-org:api:v3:errors:UpdateConflict') {
-      this.ToastService.addError({
-        message: errorResource.message,
-        type: 'error',
-        link: {
-          text: this.I18n.t('js.hal.error.update_conflict_refresh'),
-          target: () => this.apiV3Service.work_packages.id(resource).refresh(),
-        },
-      });
+      if (this.primerizedActivitiesEnabled) {
+        // currently we do not have a programmatic way to show the primer flash messages
+        // so we just do a request to the server to show it
+        // should be refactored once we have a programmatic way to show the primer flash messages!
```

Comment:
🟠 Q: Would you imagine sth similar to the `ToastService`? I reckon with the hotwire approach, it's fine to have the template server rendered either way.

Reply:
My expectation is that you would pass this state to the backend request for updating the activity, which returns a toast in case of a conflict. I briefly talked to Jonas about this and agreed that a separate request is an acceptable hack for now